app/views/layouts/application.html.haml

```diff
@@ -11,6 +11,10 @@
     = render 'layouts/navbar'
     .panel.panel-default
       .panel-body
-        %p#notice= notice
-        %p#alert= alert
+        - flash.each do |name, message|
+          .alert{class: "alert-#{name}"}
+            = message.try(:join, '. ') || message
```

Comment:
I think I'd recommend a `<ul>` here (or something). When there a larger number of errors it starts getting a little tough to parse:

![2019-11-05_14-58](https://user-images.githubusercontent.com/240275/68241573-e74d4680-ffdc-11e9-9201-df232099b206.png)